### PR TITLE
Workspace token updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated information on the workspace token on various pages, along with info on access permissions and how to obtain it.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Workspace token moved from dashboard home page to settings](https://www.notion.so/rudderstacks/Workspace-token-moved-from-dashboard-home-page-to-settings-8b6324e3318e41e49b1127bbf4528c78)

### CODE REVIEW

File: docs/faqs.mdx [hunk 1]
```diff
@@ -19,6 +19,19 @@ If you come across any issue not listed in this guide, feel free to start a conv
...
+
+The workspace token is a unique identifier associated with your current RudderStack workspace. To obtain your workspace token, go to **Settings** > **Company**, as shown:
+
+<img src="/assets/rs-cloud/workspace-token.png" alt="Workspace Token" />
+
+<div class="warningBlock">
+
+For security purposes, the workspace token is hidden, by default. You must have <Link to="/rudderstack-cloud/teammates/#admin">administrative privileges</Link> to access the token. 
+
+To view the workspace token, click on <strong>Reveal Token</strong> and enter the password associated with your RudderStack account.
```
Comment (shekhar-rudder) [inline docs/faqs.mdx:32]: Password won't be asked if the user has google login enabled. Though I don't know if we should handle it as well

Comment (ameypv-rudder) [reply docs/faqs.mdx:32]: Do you think we should include this information then? @shekhar-rudder

Comment (shekhar-rudder) [reply docs/faqs.mdx:32]: I think its okay if we don't include it as there is no extra step involved here

File: docs/faqs.mdx [hunk 2]
```diff
@@ -19,6 +19,19 @@ If you come across any issue not listed in this guide, feel free to start a conv
...
 
+#### What is a workspace token? Where do I get it?
+
+The workspace token is a unique identifier associated with your current RudderStack workspace. To obtain your workspace token, go to **Settings** > **Company**, as shown:
+
+<img src="/assets/rs-cloud/workspace-token.png" alt="Workspace Token" />
+
+<div class="warningBlock">
+
+For security purposes, the workspace token is hidden, by default. You must have <Link to="/rudderstack-cloud/teammates/#admin">administrative privileges</Link> to access the token.
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/faqs.mdx]: Added this bit.

File: docs/faqs.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/faqs.mdx:40]: Sounds good

**Review by shekhar-rudder — APPROVED** [review]

File: docs/faqs.mdx [hunk 3]
```diff
@@ -19,6 +19,26 @@ If you come across any issue not listed in this guide, feel free to start a conv
...
+The workspace token is not available for the [RudderStack Pro and Enterprise](https://www.rudderstack.com/pricing/) versions. 
+</div>
+
+To obtain your workspace token, go to **Settings** > **Company**, as shown:
+
+<img src="/assets/rs-cloud/workspace-token.png" alt="Workspace Token" />
+
+<div class="infoBlock">
+
+By default, the workspace token is hidden, for security purposes. You must have <Link to="/rudderstack-cloud/teammates/#admin">administrative privileges</Link> to access the token.
```
Comment (ishachopra0401) [inline docs/faqs.mdx:37]:
```suggestion
By default, the workspace token is hidden for security purposes. You must have <Link to="/rudderstack-cloud/teammates/#admin">administrative privileges</Link> to access the token.
```

File: docs/faqs.mdx [hunk 4]
```diff
@@ -19,6 +19,26 @@ If you come across any issue not listed in this guide, feel free to start a conv
 
 Head over to our <Link to="/get-started">Getting Started with RudderStack Cloud</Link> guide to configure your sources and destinations and start building your data pipelines.
 
+#### What is a workspace token? Where do I get it?
+
+The workspace token is a unique identifier associated with your current RudderStack workspace. 
+
+<div class="warningBlock">
+
+The workspace token is not available for the [RudderStack Pro and Enterprise](https://www.rudderstack.com/pricing/) versions.
```
Comment (ishachopra0401) [inline docs/faqs.mdx:28]: Should we mention the alternative for pro and enterprise users? or why would they not need it?

File: docs/faqs.mdx [hunk 5]
```diff
@@ -42,7 +62,7 @@ An open source data plane URL looks like `http:localhost:8080` where `8080` is t
 
 #### To get started with RudderStack on my local machine, is it mandatory to get the workspace token from RudderStack dashboard?
 
-The token (also referred to as the workspace token) is required if you are installing RudderStack in your own environment and wish to use the RudderStack-hosted control plane. It is a unique identifier for your configuration settings which RudderStack can fetch to track your instrumentations. 
+The token (also referred to as the <Link to="/rudderstack-cloud/dashboard-overview/#workspace-token">workspace token</Link>) is required if you are installing RudderStack in your own environment and wish to use the RudderStack-hosted control plane. It is a unique identifier for your configuration settings which RudderStack can fetch to track your instrumentations.
```
Comment (ishachopra0401) [inline docs/faqs.mdx:65]:
```suggestion
The <Link to="/rudderstack-cloud/dashboard-overview/#workspace-token">workspace token</Link> is required if you are installing RudderStack in your own environment and wish to use the RudderStack-hosted control plane. It is a unique identifier for your configuration settings which RudderStack can fetch to track your instrumentations. 
```

File: docs/faqs.mdx [hunk 6]
```diff
@@ -205,17 +225,19 @@ Unfortunately, your workspace name is not changeable currently. We are planning
 
 #### What is the difference between source write key and the workspace token?
 
-The write key is different from your workspace token. The write key is associated with the source, while the workspace token is associated with your RudderStack workspace, as shown:
+The write key is different from your workspace token. The write key is associated with the source, while the <Link to="/rudderstack-cloud/dashboard-overview/#workspace-token">workspace token</Link> is associated with your RudderStack workspace, as shown:
+
+<img src="../assets/rs-cloud/workspace-token.png" alt="Workspace Token" />
```
Comment (ishachopra0401) [inline docs/faqs.mdx:230]: Should we also add a screenshot for source write key? Currently, there is no reference link/location/screenshot for write key.

File: docs/get-started/glossary.mdx [hunk 7]
```diff
@@ -204,18 +204,34 @@ When sending your events to a data warehouse via RudderStack, you don't need to
...
 
-## Token
+## Workspace Token
 
-The token (also referred to as the workspace token) is a unique identifier of your RudderStack workspace. You can find it by logging in to your [RudderStack dashboard](https://app.rudderstack.com/).
+The workspace token uniquely identifies your RudderStack workspace. 
+
+<div class="warningBlock">
+
+The workspace token is not available for the [RudderStack Pro and Enterprise](https://www.rudderstack.com/pricing/) versions.
```
Comment (ishachopra0401) [inline docs/get-started/glossary.mdx:213]: same as above

File: docs/get-started/glossary.mdx [hunk 8]
```diff
@@ -204,18 +204,34 @@ When sending your events to a data warehouse via RudderStack, you don't need to
...
+The workspace token is not available for the [RudderStack Pro and Enterprise](https://www.rudderstack.com/pricing/) versions. 
+</div>
+
+To get the workspace token, log into your [RudderStack dashboard](https://app.rudderstack.com/) and go to **Settings** > **Company**, as shown:
+
+<img src="../assets/rs-cloud/workspace-token.png" alt="Workspace Token" />
+
+<div class="warningBlock">
+
+By default, the workspace token is hidden, for security purposes. You must have <Link to="/rudderstack-cloud/teammates/#admin">administrative privileges</Link> to access the token.
```
Comment (ishachopra0401) [inline docs/get-started/glossary.mdx:222]:
```suggestion
By default, the workspace token is hidden for security purposes. You must have <Link to="/rudderstack-cloud/teammates/#admin">administrative privileges</Link> to access the token.
```

File: docs/rudderstack-cloud/dashboard-overview.mdx [hunk 9]
```diff
@@ -171,6 +165,26 @@ Refer to the <a href="https://www.rudderstack.com/docs/rudderstack-api/personal-
...
+The workspace token is not available for the [RudderStack Pro and Enterprise](https://www.rudderstack.com/pricing/) versions. 
+</div>
+
+To get your workspace token, go to **Settings** > **Company**, as shown:
+
+<img src="../assets/rs-cloud/workspace-token.png" alt="Workspace Token" />
+
+<div class="warningBlock">
+
+By default, the workspace token is hidden, for security purposes. You must have <Link to="/rudderstack-cloud/teammates/#admin">administrative privileges</Link> to access the token.
```
Comment (ishachopra0401) [inline docs/rudderstack-cloud/dashboard-overview.mdx:183]:
```suggestion
By default, the workspace token is hidden for security purposes. You must have <Link to="/rudderstack-cloud/teammates/#admin">administrative privileges</Link> to access the token.
```

File: docs/rudderstack-open-source/installing-and-setting-up-rudderstack/kubernetes.mdx [hunk 10]
```diff
@@ -25,13 +25,13 @@ You can find the Helm chart in the [RudderStack GitHub repository](https://githu
 If you're using the RudderStack-hosted control plane, follow these steps to set up the data plane on your Kubernetes cluster:
 
 1. Log into your [RudderStack Open Source dashboard](https://app.rudderstack.com/signup?type=opensource).
-2. Note and copy your workspace token from the top of the page, as shown:
+2. Copy your workspace token by going to **Settings** > **Company**, as shown:
```
Comment (ishachopra0401) [inline docs/rudderstack-open-source/installing-and-setting-up-rudderstack/kubernetes.mdx:28]:
```suggestion
2. Copy your workspace token from **Settings** > **Company**, as shown:
```

File: docs/rudderstack-open-source/installing-and-setting-up-rudderstack/docker.mdx [hunk 11]
```diff
@@ -18,13 +18,13 @@ The Docker setup is the easiest and the fastest way to get up and running with R
 If you're using a RudderStack-hosted control plane, follow these steps to set up the data plane in your Docker environment:
 
 1. Log into your [RudderStack Open Source dashboard](https://app.rudderstack.com/signup?type=opensource).
-2. Note and copy your workspace token from the top of the page, as shown:
+2. Copy your workspace token by going to **Settings** > **Company**, as shown:
```
Comment (ishachopra0401) [inline docs/rudderstack-open-source/installing-and-setting-up-rudderstack/docker.mdx:21]:
```suggestion
2. Copy your workspace token from **Settings** > **Company**, as shown:
```

File: docs/rudderstack-open-source/installing-and-setting-up-rudderstack/developer-machine-setup.mdx [hunk 12]
```diff
@@ -21,13 +21,13 @@ To set up RudderStack, you will need to set up and install the following tools:
 If you're using a RudderStack-hosted control plane, follow these steps to set up the data plane in your development environment:
 
 1. Log into your [RudderStack Open Source dashboard](https://app.rudderstack.com/signup?type=opensource).
-2. Note and copy your workspace token from the top of the page, as shown:
+2. Copy your workspace token by going to **Settings** > **Company**, as shown:
```
Comment (ishachopra0401) [inline docs/rudderstack-open-source/installing-and-setting-up-rudderstack/developer-machine-setup.mdx:24]:
```suggestion
2. Copy your workspace token from **Settings** > **Company**, as shown:
```

File: docs/faqs.mdx (hunk 4 above)
Comment (ameypv-rudder) [reply docs/faqs.mdx:28]: We handle their pro/enterprise deployment so they don't need it.

File: docs/faqs.mdx [hunk 13]
```diff
@@ -19,6 +19,26 @@ If you come across any issue not listed in this guide, feel free to start a conv
...
+The workspace token is a unique identifier associated with your current RudderStack workspace. 
+
+<div class="warningBlock">
+
+The workspace token is not available for the [RudderStack Pro and Enterprise](https://www.rudderstack.com/pricing/) versions. 
+</div>
+
+To obtain your workspace token, go to **Settings** > **Company**, as shown:
+
+<img src="/assets/rs-cloud/workspace-token.png" alt="Workspace Token" />
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/faqs.mdx:33]: yep, you are :P just pinged on Slack.

File: docs/faqs.mdx (hunk 4 above)
Comment (ishachopra0401) [reply docs/faqs.mdx:28]: So would this make sense?

The workspace token is not available for the RudderStack Pro and Enterprise users as RudderStack handles their deployment.

Comment (ameypv-rudder) [reply docs/faqs.mdx]: Yep - tweaked it slightly. Let me know if this makes sense.

File: docs/faqs.mdx (hunk 6 above)
Comment (ameypv-rudder) [reply docs/faqs.mdx]: Added screenshot for the write key.

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**